examples/golang/actors/http-hello-world/wit/deps.toml

```diff
@@ -1,2 +1,4 @@
 http = "https://github.com/WebAssembly/wasi-http/archive/v0.2.0-rc-2023-12-05.tar.gz"
-
+io = "https://github.com/WebAssembly/wasi-io/archive/v0.2.0-rc-2023-11-10.tar.gz"
+logging = "https://github.com/WebAssembly/wasi-logging/archive/main.tar.gz"
+keyvalue = "https://github.com/WebAssembly/wasi-keyvalue/archive/main.tar.gz"
```

Comment:
Are these being at main at risk of floating? Should they be versioned instead like the others?

Reply:
Good question. No risk as long as you don't update of course which the tutorial doesn't say to do. This is the reference that we use in the actor and runtime crate 🤔 I just worked with that precedent

Reply:
Sounds good to me